.github/workflows/detect_version_bump.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Create Release
         # Using full SHA for security
         # https://docs.github.com/en/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions
-        uses: ncipollo/release-action@4c75f0f2e4ae5f3c807cf0904605408e319dcaac
+        uses: ncipollo/release-action@4c75f0f2e4ae5f3c807cf0904605408e319dcaac # renovate: tag=v1.11.1
```

Comment:
For any future reviewers, there are docs on how this comment works [here](https://docs.renovatebot.com/modules/manager/github-actions/#additional-information).